package.json

```diff
@@ -1,27 +1,36 @@
 {
-  "name": "dbload-example-app",
+  "name": "dbload-example-app2",
   "version": "1.0.0",
   "main": "server/server.js",
   "scripts": {
     "pretest": "jshint ."
   },
   "dependencies": {
-    "async": "^0.9.0",
+    "async": "^1.4.2",
     "compression": "^1.0.3",
+    "cors": "^2.5.2",
     "errorhandler": "^1.1.1",
-    "loopback": "^2.8.0",
-    "loopback-boot": "^2.4.0",
-    "loopback-datasource-juggler": "^2.7.0",
-    "restler": "^3.2.2",
-    "serve-favicon": "^2.0.1"
+    "fs": "0.0.2",
+    "loopback": "^2.14.0",
+    "loopback-boot": "^2.6.5",
+    "loopback-connector-postgresql": "^2.3.1",
+    "loopback-connector-sqlite3": "^1.0.0",
+    "loopback-datasource-juggler": "^2.19.0",
+    "memcached": "^2.1.0",
```

Comment:
shouldn't you test both the underlying DBs, and the loopback connector for the DB (when available)? mysql, pg, mongo, oracle connectors are missing, as well as mssql fin raw and connector form.

Maybe its too much work for now, or hitting every probe is not the point, if so, that's fine.

There are other probes that might not be hit, that aren't DBs, list of all probed modules are here:

- https://github.com/strongloop/strongops/tree/master/lib/probes

probably this app is only to generate db metrics, though, but its missing some of https://github.com/strongloop/strongops/blob/master/README.md#third-party-module-metrics:

- leveldown
- riak-js
- memcache (you can use memcache and memcached in the same app... they should be counted in the same metrics name)